test/tools/cli/IonProcess.java

```diff
@@ -630,37 +593,33 @@ private static Event ionStreamToEvent(EventType eventType,
         SymbolToken fieldName = ionReader.getFieldNameSymbol();
         SymbolToken[] annotations = ionReader.getTypeAnnotationSymbols();
 
-        String valueText = null;
-        byte[] valueBinary = null;
+        IonValue value = null;
         if (eventType == EventType.SCALAR) {
             try (
                     ByteArrayOutputStream textOut = new ByteArrayOutputStream();
-                    IonWriter textWriter = IonTextWriterBuilder.standard().build(textOut);
-                    ByteArrayOutputStream binaryOut = new ByteArrayOutputStream();
-                    IonWriter binaryWriter = IonBinaryWriterBuilder.standard().build(binaryOut);
-                    ) {
+                    IonWriter tempWriter = IonTextWriterBuilder.standard().build(textOut);
```

Comment:
One thing we can do is create a constant `IonTextWriterBuilder` instance and reuse it whenever we need a temporary text writer. The builder just holds configuration, so it's reusable. This will improve performance.

Reply:
Added.